client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx

```diff
@@ -43,73 +55,111 @@ export default function ItemPreviewPaneHeader( {
 	const siteIconFallback =
 		extraProps?.siteIconFallback ?? ( itemData.isDotcomSite ? 'wordpress-logo' : 'color' );
 
+	const handlePhpVersionClick = () => {
+		page( `/hosting-config/${ selectedSite?.domain }#php` );
+	};
+
+	const handleWpVersionClick = () => {
+		page( `/hosting-config/${ selectedSite?.domain }#wp` );
```

Comment:
Probably not a blocker, but I'm wondering if we want to handle this link differently for simple sites. Or perhaps not render it as a link at all?

Because there is no `/hosting-config` for a simple site, clicking the link lands the user on `/hosting-features` with an upgrade nudge. That's not inherently bad, but I wonder if it might be jarring for the user.

Someone clicking on that link is likely hoping to learn more about their current version of WordPress but instead they see an upgrade button. They might be looking to switch versions, but the upgrade screen doesn't make any mention of what those options are post-upgrade.